1-org/modules/network/variables.tf

```diff
@@ -45,6 +45,13 @@ variable "env_code" {
   description = "A short form of the environment to prepare within the Google Cloud organization (ex. d)."
 }
 
+variable "vpc" {
+  description = "The type of VPC to attach the project to. Possible options are base or restricted."
```

Comment:
Since you are explicit saying that the possible values for this variable are just `base` or `restricted`, I would recommend it to add a [custom validation](https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules) to enforces that.

In addition to that, related to this "null" string, if this is really the text we want to add for projects that doesn't have VPC, we need to list it as a possible/valid string input option for the variable too.